ronn/geopm_report.7.ronn

```diff
@@ -0,0 +1,288 @@
+                           The region hint is determined by the hint
+                           passes to `geopm_region()` for the region
+                           and any nested regions.
+
+  `time-hint-memory (s)`: The portion of sync-runtime where the region
+                          hint was GEOPM_REGION_HINT_MEMORY.  The
+                          region hint is determined by the hint passes
+                          to `geopm_region()` for the region and any
+                          nested regions.
+
+  `time-hint-io (s)`: The portion of sync-runtime where the region
+                      hint was GEOPM_REGION_HINT_IO.  The region hint
+                      is determined by the hint passes to
+                      `geopm_region()` for the region and any nested
+                      regions.
+
+  `time-hint-serial (s)`: The portion of sync-runtime where the region
+                          hint was GEOPM_REGION_HINT_SERIAL.  The
+                          region hint is determined by the hint passes
+                          to `geopm_region()` for the region and any
+                          nested regions.
+
+  `time-hint-parallel (s)`: The portion of sync-runtime where the
+                            region hint was
+                            GEOPM_REGION_HINT_PARALLEL.  The region
+                            hint is determined by the hint passes to
+                            `geopm_region()` for the region and any
+                            nested regions.
+
+  `time-hint-unknown (s)`: The portion of sync-runtime where the
+                           region hint was GEOPM_REGION_HINT_UNKNOWN.
+                           The region hint is determined by the hint
+                           passes to `geopm_region()` for the region
+                           and any nested regions.
+
+  `time-hint-unset (s)`: The portion of sync-runtime where the region
+                         hint was GEOPM_REGION_HINT_UNSET.  The region
+                         hint is unset when exiting a region
+                         (i.e. when a CPU is in the unmarked region).
+
+* `Report Extensions`:
+
+The report can be extended by agents, or by through the
+`--geopm-report-signals` option to `geopmlaunch` which corresponds to
+the `GEOPM_REPORT_SIGNALS` environment variable.  See the description
+for `--geopm-report-signals` in **geopmlaunch(1)** for more details.
+Signals in the `GEOPM_REPORT_SIGNALS` list will be added to individual
+regions, the epoch totals, and the application totals for each host.
+
+Agents can add keys to the report header, host section, or individual
+region sections using the `report_header()`, `report_host()` or
+`report_region()` methods respectively.  See **geopm::Agent(3)** for
+more information about the report extensions available to agents.
+
+
+## EXAMPLES
+
+Suppose we have a system with 2 sockets per node and 2 CPUs per packge
+(socket) (assume 1 CPU per core for simplicity).  The application
```

Comment:
assume 1 CPU per core for simplicity, i.e. hyper-threading is turned off if applicable